### PR TITLE
Render changes matt

### PR DESCRIPTION
**Pull request by mnolan34**
Will have to rebase it after request

### CODE REVIEW

**Review by kachalmers — COMMENTED** [review]
For the user to more easily understand what's happening after they click the update button,  I'd recommend turning the Update button into something like this:
`
           <button
                onClick = {()=> {
                    updateTuit(tid, {...tuit, tuit: newTuitString});
                    window.location.reload();
                    alert("Tuit has been updated!");
                }}
                className = "float-end tuit-button me-1">
                Update
            </button>
`
This would update the tuit, refresh the page to show the new "old tuit", and send an alert to the user to tell them the tuit was updated.

File: src/components/explore/tuits-by-tag.js [hunk 1]
```diff
@@ -29,19 +30,30 @@ const TuitsByTag = () => {
...
         tagService.findTuitsWithTag(tempTag)
             .then(newTuits => {
                 setTuits(newTuits);
             });
-        console.log("Tuits with " + tempTag + " added!");
+        //console.log("Tuits with " + tempTag + " added!");
         //console.log(tuits.length);
         navigate(tempTag);
     }
+    /*
```
Comment (kachalmers) [inline src/components/explore/tuits-by-tag.js:48]: Do we want to keep these comments in the final code?

File: src/components/explore/tuits-by-tag.js [hunk 2]
```diff
@@ -29,19 +30,30 @@ const TuitsByTag = () => {
...
+        console.log(newOrder);
+        setOrder(newOrder);
+        //sortTuits();
+        //findTuitsWithTag();
+    }
+     */
+
 
     const findTuitsWithTag = () => {
         // #KAC
```
Comment (kachalmers) [inline src/components/explore/tuits-by-tag.js:60]: Do we want to keep these comments in the final code?

File: src/components/tuits/update-tuit.js [hunk 3]
```diff
@@ -15,6 +15,13 @@ const TuitScreen = () => {
     return(
         <div>
 
+            <h4>Old Tuit:</h4>
+            <a>{tuit.tuit}</a>
+
+            <br/>
+            <br/>
+
```
Comment (mcartervt) [inline src/components/tuits/update-tuit.js:23]: This functionality is working for me.  I agree with Katie C., let's add the alert idea.

**Review by mcartervt — CHANGES_REQUESTED** [review]
Functionality works for me.  Would like the conflicts resolved before approval.  Also agree with Katie C. about the alert idea.

**Comment by kachalmers** [comment]
> For the user to more easily understand what's happening after they click the update button, I'd recommend turning the Update button into something like this: `<button onClick = {()=> { updateTuit(tid, {...tuit, tuit: newTuitString}); window.location.reload(); alert("Tuit has been updated!"); }} className = "float-end tuit-button me-1"> Update </button>` This would update the tuit, refresh the page to show the new "old tuit", and send an alert to the user to tell them the tuit was updated.

(Option 2)
Go back to previous page when tuit is updated. See file from microsoft teams chat for full file implementation.
`
            <button
                onClick = {()=> {
                    updateTuit(tid, {...tuit, tuit: newTuitString});
                    navigate(-1);   // go back to previous page
                    alert("Tuit has been updated!");
                }}
                className = "float-end tuit-button me-1">
                Update
            </button>
`

File: src/components/tuits/update-tuit.js (hunk 3 above)
Comment (mnolan34) [reply src/components/tuits/update-tuit.js:23]: Alert idea implemented

**Comment by mnolan34** [comment]
> > For the user to more easily understand what's happening after they click the update button, I'd recommend turning the Update button into something like this: `<button onClick = {()=> { updateTuit(tid, {...tuit, tuit: newTuitString}); window.location.reload(); alert("Tuit has been updated!"); }} className = "float-end tuit-button me-1"> Update </button>` This would update the tuit, refresh the page to show the new "old tuit", and send an alert to the user to tell them the tuit was updated.
> 
> (Option 2) Go back to previous page when tuit is updated. See file from microsoft teams chat for full file implementation. `<button onClick = {()=> { updateTuit(tid, {...tuit, tuit: newTuitString}); navigate(-1); // go back to previous page alert("Tuit has been updated!"); }} className = "float-end tuit-button me-1"> Update </button>`

This is cool, implemented in the code, and will begin testing

**Review by kachalmers — APPROVED** [review]
Everything functionally works for me with the main branch updates and the new updates from this branch.

File: src/components/explore/tuits-by-tag.js (hunk 1 above)
Comment (mcartervt) [reply src/components/explore/tuits-by-tag.js:48]: We can always do a "Clean Up" branch after.

File: src/components/tuits/update-tuit.js (hunk 3 above)
Comment (mcartervt) [reply src/components/tuits/update-tuit.js:23]: Woot!

**Review by mcartervt — APPROVED** [review]
Works like a charm!  

I think we can leave some of the cleanup to its own branch after this.  

Approved!

File: src/components/explore/tuits-by-tag.js (hunk 1 above)
Comment (kachalmers) [reply src/components/explore/tuits-by-tag.js:48]: Resolved in https://github.com/kachalmers/software-engineering-react-s2-team1-project/pull/22 .

File: src/components/explore/tuits-by-tag.js (hunk 2 above)
Comment (kachalmers) [reply src/components/explore/tuits-by-tag.js:60]: Resolved in https://github.com/kachalmers/software-engineering-react-s2-team1-project/pull/22 .

**Pull request merged**